src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -43,21 +41,67 @@
   [route]
   (money/wei->ether (reduce money/add (map calculate-gas-fee route))))
 
+(defn- path-amount-in
+  [path]
+  (-> path :amount-in money/from-hex))
+
+(defn- path-amount-out
+  [path]
+  (if (-> path :bridge-name (= constants/bridge-name-hop))
```

Comment:
[Opinion]

Thread macro feels redundant:
`(= (:bridge-name path) constnats/bridge-name-hop)` is more clear.